src/Sylius/Behat/Context/Api/Shop/ChannelContext.php

```diff
@@ -57,4 +57,36 @@ public function iShouldShopUsingTheCurrency(string $currencyCode): void
             sprintf('%s/shop/currencies/%s', $this->apiUrlPrefix, $currencyCode),
         );
     }
+
+    /**
+     * @Then I should be able to shop using the :currencyCode currency
+     */
+    public function iShouldBeAbleToShopUsingTheCurrency(string $currencyCode): void
+    {
+        $this->client->index(Resources::CURRENCIES);
```

Comment:
I wondered if we should check the "currencies" key on the `channels/{id}` endpoint, but it seems that both return the same data (except that the channel contains IRI)